crates/ironrdp-cliprdr-format/src/bitmap.rs

```diff
@@ -144,14 +144,14 @@ impl PduEncode for CiexyzTriple {
 /// We don't use the optional `bmiColors` field, because it is only relevant for bitmaps with
 /// bpp < 24, which are not supported yet, therefore only fixed part of the header is implemented.
 ///
-/// INVARIANT: `width` and `height` magnitudes are less than or equal to `u16::MAX`.
-/// INVARIANT: `bit_count` is less than or equal to `32`.
-///
 /// [BITMAPINFO]: https://learn.microsoft.com/en-us/windows/win32/api/wingdi/ns-wingdi-bitmapinfo
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
 struct BitmapInfoHeader {
+    /// INVARIANT: `width.abs() <= 10_000`
     width: i32,
+    /// INVARIANT: `height.abs() <= 10_000`
```

Comment:
Smaller bounds are helping with proving relations such as
```
height * width * n_samples <= 10_000 * 10_000 * 4 < u32::MAX
```
And a 10000x10000 bitmap is a _very_ big image to transfer through the clipboard already.

Reply:
Makes sense, agreed